.github/workflows/chromatic.yml

```diff
@@ -40,6 +40,8 @@ jobs:
               - 'packages/components/**'
             studio:
               - 'apps/studio/**'
+              - 'packages/components/src/interfaces/**'
+              - 'packages/components/src/types/**'
```

Comment:
curious - why is there a need to add this?

Reply:
Context: https://opengovproducts.slack.com/archives/C06R4DX966P/p1732607260729969?thread_ts=1732597220.921359&cid=C06R4DX966P